tests/performance_tests/performance_test_basic.py

```diff
@@ -95,8 +88,12 @@ class SpecifiedContract:
         _validationNodeIds: list = field(default_factory=list)
         _apiNodeIds: list = field(default_factory=list)
         nonProdsEosVmOcEnable: bool = False
+        _killEosInstances: bool = True
+        _killWallet: bool = True
 
         def __post_init__(self):
+            self._killEosInstances = not self.dontKill
+            self._killWallet = not self.dontKill
```

Comment:
As these (`_killEosInstances` and `_killWallet`) have been removed from `TestHelper`, instead of moving these to `ClusterConfig` I think they should be removed here as well as they are no longer used.

Reply:
Good point.  Removed.

Reply:
12706fd3